test/openAPI/features/support/wallet_generate_link_code.js

```diff
@@ -140,19 +154,150 @@ When(
 
 // Scenario: Not able to generate link code because of reause of the completed transaction_id
 // Given and others Then for this scenario are written in the aforementioned example
+Given('The first authorization flow for transactionId ends', async () => {
+  specWalletGenerateLinkCode
+    .post(baseUrl)
+    .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+    .withCookies(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+    .withJson({
+      requestTime: new Date().toISOString(),
+      request: {
+        transactionId: 'transaction_id_003',
+      },
+    });
+
+  await specWalletGenerateLinkCode.toss();
+
+  await specWalletLinkTransaction
+    .post(localhost + walletLinkTransactionEndpoint)
+    .withJson({
+      requestTime: new Date().toISOString(),
+      request: {
+        linkCode: specWalletGenerateLinkCode._response.json.response.linkCode,
+      },
+    });
+
+  await specWalletLinkTransaction.toss();
+
+  await specWalletConsent.post(localhost + walletConsentEndpoint).withJson({
```

Comment:
I am not sure if I am following the code correctly, but after the link transaction, we have to authenticate before consent